src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -1081,3 +1081,9 @@ The Split integration is only available for organizations on the Business and En
 
 Follow the instructions in the link below:  
 <https://help.split.io/hc/en-us/articles/360029879431>
+
+## Troubleshooting
+If you're having trouble setting up Sentry with your on-premise integration like JIRA Server, Bitbucket Server, Github Enterprise or Gitlab:
+- Ensure that the installation URL you provided is a fully qualified domain name (FQDN) which is resolvable on the internet.
```

Comment:
@thinkocapo it would be great if you could post a link that could check if a URL is an FQDN. But I should mention this might change with a PR that is open: https://github.com/getsentry/sentry/pull/17404

Reply:
@scefali would it be safer to avoid using the 'FQDN' term here then since the status of getsentry/sentry#17404 is unknown?

@jonesphillip do you follow what this is about?

Reply:
I'm unsure what call to make on this here. What should we write?

Reply:
@thinkocapo @markstory This is referring to this issue. https://github.com/getsentry/sentry/issues/17376

This wouldn't work as sentry can't talk to wouldn't be able to talk `https://bitbucket` correct?